### DISCUSSION TITLE
Ring keypad brightness and volume

### DISCUSSION THREAD

**Discussion by JeremyLaurenson** [dgreif/ring#1122] [Q&A] [answered]
I am using the ring homebridge plug-in, and every so often all of the keypad brightness turn to zero and sometimes their volumes are turned down. 

I am not sure if I should contact Ring or if this is somehow something to do with the plug-in.

Does anybody have experience with something like this happening, is there an API that the plug-in might be hitting or should I contact Ring?

**Comment by dgreif** [comment, marked as answer]
It's possible this is coming from the homebridge plugin. You could get into this scenario if you, for example, told Siri to "turn off all the lights". Since the keypad is technically a "light" accessory, she would turn it off. If you don't care to control these devices with HomeKit, you can hide them with the `hideDeviceIds` option
